examples/chip-tool/commands/discover/DiscoverCommissionablesCommand.cpp

```diff
@@ -24,54 +24,60 @@ using namespace ::chip;
 
 void DiscoverCommissionablesCommandBase::OnDiscoveredDevice(const chip::Dnssd::DiscoveredNodeData & nodeData)
 {
-    nodeData.LogDetail();
```

Comment:
I guess my question is what happens if we're using minimal mdns here, not platform.  Does that log too?  Would prefer we keep logging here, but a followup for that is fine.

Reply:
I have added it back. I tried with minimal and obviously nothing is logged. I will do something to avoid the double logging in a followup.